railties/lib/rails/application/configuration.rb

```diff
@@ -79,6 +79,7 @@ def load_defaults(target_version)
 
           if respond_to?(:action_view)
             action_view.form_with_generates_remote_forms = true
+            action_view.form_with_skip_default_ids = false
```

Comment:
We need to keep it `true` since we are not going to change the current behavior in 5.1.